tests/configurations/integration/jest.config.js

```diff
@@ -2,20 +2,29 @@ module.exports = {
   transform: {
     '^.+\\.ts$': 'ts-jest',
   },
-  coverageReporters: ['text', 'html'],
-  rootDir: '../../../.',
   testMatch: ['<rootDir>/tests/integration/**/*.spec.ts'],
-  setupFiles: ['<rootDir>/tests/configurations/jest.setup.js'],
+  coverageReporters: ['text', 'html'],
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.ts', '!*/node_modules/', '!/vendor/**', '!*/common/**', '!**/models/**', '!<rootDir>/src/*'],
```

Comment:
models should be inside the coverage report. also ignore specific files and not the entire base folder.